core/src/main/java/ch/cyberduck/core/Profile.java

```diff
@@ -570,15 +576,17 @@ public boolean isUTCTimezone() {
     }
 
     private String value(final String key) {
-        return dict.stringForKey(key);
+        return substitutor.replace(dict.stringForKey(key));
     }
 
     private List<String> list(final String key) {
         final List<String> list = dict.listForKey(key);
         if(null == list) {
             return Collections.emptyList();
         }
-        return list;
+        final ArrayList<String> substituted = new ArrayList<>(list);
+        substituted.replaceAll(substitutor::replace);
+        return substituted;
     }
 
     private boolean bool(final String key) {
```

Comment:
no support for injecting `bool`s?

Reply:
Not possible as the property list dtd does not allow arbitrary values in this case.